File: backend-module/eneity/AIComment.java
@Data
public class AIComment {
    private int id;
    private int studentId;
    private int teacherId;
    private String semester;
    private String content;
    private int isTeacherEdited;
    private String status;
    private String generatedBy;
    private int tokensUsed;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
    private boolean isDeleted;
}

File: frontend/src/types/aiComment.ts
export interface AiComment {
  id: number
  studentId: number
  teacherId: number
  semester: string
  content: string
  isTeacherEdited: number
  status: string
  generatedBy: string
  tokensUsed: number
  createdAt: string
  updatedAt: string
  isDeleted: boolean
}

export interface AiCommentVersion {
  id: number
  commentId: number
  versionNo: number
  content: string
  source: string
  tokensUsed: number
  createdAt: string
  isDeleted: boolean
}

export interface AiCommentGenerateForm {
  studentIds: number[]
  semester: string
}
